### PR TITLE
Improvments for levels

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: app/controllers/admin/profiles_controller.rb [hunk 1]
```diff
@@ -19,6 +19,8 @@ def show
     def change_state
       @profile = Profile.find(params[:id])
       if @profile.update(state: params[:state])
+        account = Account.find(@profile.account_id)
```
Comment (mod) [inline app/controllers/admin/profiles_controller.rb]: yes, but it doesn't verify that state == :approved 
write 

remove
```
account = Account.find(@profile.account_id)
```
use
```
@profile.account.set_level(:identity) if @profile.state == :approved
```

File: app/models/account.rb [hunk 2]
```diff
@@ -18,6 +18,8 @@ def role
   end
 
   def increase_level
```
Comment (mod) [inline app/models/account.rb]: make a switch case
```ruby
def set_level(step)
  case(step)
  when :mail
    self.level = 1
  when :phone
    self.level = 2
  when :identity
    self.level = 3
  when :address
    self.level = 4
  end
end
```

File: app/controllers/phones_controller.rb [hunk 3]
```diff
@@ -3,7 +3,11 @@
 class PhonesController < ApplicationController
 
   def new
-    @phone = Phone.new
+    if current_account.level != 1
```
Comment (gfedorenko) [inline app/controllers/phones_controller.rb]: not != but <. Cause person with level 2 can add another phone

Comment (vpetrusenko) [reply app/controllers/phones_controller.rb]: oh okay, sorry

**Pull request merged**